src/testapp/sessions/vehicles_machines.cpp

```diff
@@ -234,24 +234,26 @@ Session setup_thrust_indicators(
         ActiveEnt const     partEnt     = rScnParts.partToActive[part];
 
         for (MachinePair const pair : rScnParts.partToMachines[part])
-        if (pair.type == gc_mtMagicRocket)
         {
-            DrawEnt const   drawEnt         = rThrustIndicator.rktToDrawEnt[pair.local];
-            MachAnyId const anyId           = rockets.localToAny[pair.local];
-
-            auto const&     portSpan        = floats.machToNode[anyId];
-            NodeId const    throttleIn      = connected_node(portSpan, ports_magicrocket::gc_throttleIn.port);
-            NodeId const    multiplierIn    = connected_node(portSpan, ports_magicrocket::gc_multiplierIn.port);
-
-            float const     throttle        = std::clamp(rSigValFloat[throttleIn], 0.0f, 1.0f);
-            float const     multiplier      = rSigValFloat[multiplierIn];
-            float const     thrustMag       = throttle * multiplier;
-
-            rCtxScnRdr.m_drawTransform[drawEnt]
-                    = drawTf
-                    * Matrix4::scaling({1.0f, 1.0f, thrustMag * rThrustIndicator.indicatorScale})
-                    * Matrix4::translation({0.0f, 0.0f, -1.0f})
-                    * Matrix4::scaling({0.2f, 0.2f, 1.0f});
+          if (pair.type == gc_mtMagicRocket)
```

Comment:
accidental 2 space indent?

I've been doing this weird `for (...) if` here and there, as I think the slightly reduced nesting is worth it for weirdness as a tradeoff.

Reply:
Hrmm, alright, i can do that.